yarn.lock

```diff
@@ -3392,8 +3498,9 @@
     "@crypto-com/chain-jslib" "0.0.19"
     "@ethereumjs/common" "^2.6.2"
     "@ethereumjs/tx" "^3.5.0"
+    "@keplr-wallet/cosmos" "^0.9.16"
     "@ledgerhq/compressjs" "1.3.2"
-    "@ledgerhq/cryptoassets" "6.27.0"
+    "@ledgerhq/cryptoassets" "file:../../../Library/Caches/Yarn/v6/npm-@ledgerhq-live-common-21.36.2-8b64b83f-5ea1-46c2-aedb-01c8b75fbd46-1650460183999/node_modules/@ledgerhq/live-common/.yalc/@ledgerhq/cryptoassets"
```

Comment:
If you also depend on a LedgerJS package being modified things get more hairy indeed. You can release a tagged version of the package or get someone else to do it for you, otherwise we have the same issue as above, it will not create a build. And won't be easily testable.